lib/incito-browser/utils.js

```diff
@@ -2,16 +2,24 @@ export const formatUnit = (unit) => {
     if (unit == null) {
         return 0;
     } else if (typeof unit === 'number') {
-        return `${unit}px`;
+        return unit + 'px';
     } else if (typeof unit === 'string') {
         return unit.replace('dp', 'px');
     } else {
         return 0;
     }
 };
 
+export const escapeAttrValue = (value) => {
+    if (typeof value === 'string') {
+        return value.replace(/"/g, '&quot;');
+    }
+
+    return value;
+};
+
 export const isDefinedStr = (value) =>
-    typeof value === 'string' && value.length > 0;
+    value && typeof value === 'string' && value.length > 0;
```

Comment:
In what case is a `string` longer than 0 falsy?

Reply:
True - will remove.